README.md

```diff
@@ -1 +1,45 @@
-# my-portfolio
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My Portfolio
+
+Setting up a personal portfolio
```

Comment:
- [ ] This is a great start with the project summary. Add a little more detail to include every feature you implemented.